esmvaltool/diag_scripts/seaborn_diag.py

```diff
@@ -171,7 +174,10 @@ def _create_plot(
                 getattr(plot_obj, func_name)(func_args)
     if cfg['suptitle'] is not None:
         logger.debug("Setting `suptitle='%s'`", cfg['suptitle'])
-        plt.suptitle(cfg['suptitle'], y=1.05)
+        if plot_func_str in ['jointplot']:
+            plt.suptitle(cfg['suptitle'].format(data_frame["shape_id"].unique()[0]), y=1.05)
```

Comment:
This will cause the diagnostic to fail if no `shape_id` is present. Is this change necessary because you created a multi-panel plot (as far as I understand, this is not possible with `jointplot`)? If you need such a change, it would be better to hardcode the format string into the recipe Like `Plot of {shape_id}` and then let the diagnostic fill this. This will be much more general.